### PR TITLE
add missing types for recommended addon-centered imports

### PR DESCRIPTION
**Pull request by hipstersmoothie**
# What I did

When using the centered addon a user previously had to define their own type def. It looks like those typedefs exist, just in the dist folder. This PR just addes `.d.ts` files that re-export the build output

## How to test

- Is this testable with Jest or Chromatic screenshots? no
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

**Comment by hipstersmoothie** [comment]
@ndelangen could you take a look at this one?

File: addons/centered/ember.d.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import centered from './dist/ember';
```
Comment (ndelangen) [inline addons/centered/ember.d.ts]: These files need to be added to the files allow-list in package.json

Comment (shilman) [reply addons/centered/ember.d.ts]: why are we disallowing them in the first place? is an optimization? seems like it's going to cause a bunch of headaches. are there other benefits?

Comment (hipstersmoothie) [reply addons/centered/ember.d.ts]: `@storybook/addon-centered` doesn't actually uses "files" so I think all the files are already included

Comment (ndelangen) [reply addons/centered/ember.d.ts]: @shilman there are 2 ways to make npm not publish everything:
- explicit not-allowed list
- explicit allowed list

It's true that an allow-list is slightly more cumbersome but:
https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d

.npmignore overrides the .gitignore, which means they have be kept in sync now.

I think having an allow list is slightly more robust.

I'll work on the publishing test I mentioned hopefully soon, so we can be sure the packages work after publishing.

Comment (ndelangen) [reply addons/centered/ember.d.ts]: @hipstersmoothie it should though?

Comment (hipstersmoothie) [reply addons/centered/ember.d.ts]: I've never used npm ignores so it seems good not to from that article. I looked again and it seems there is a file list already so I could just add `*.d.ts`

Comment (shilman) [reply addons/centered/ember.d.ts]: why don't we want to publish everything? is an optimization? seems like it's going to cause a bunch of headaches.

Comment (ndelangen) [reply addons/centered/ember.d.ts]: We want to publish the minimum possible to optimise people's install-time and disk-usage.

Comment (shilman) [reply addons/centered/ember.d.ts]: How much time/space does it save?

**Comment by hipstersmoothie** [comment]
@ndelangen changed the files glob to accommodate my changes

**Review by ndelangen — APPROVED** [review]

**Pull request merged**